general_ledger/templates/header_macro.html

```diff
@@ -51,7 +51,8 @@ <h1 style="font-size:20px">{{ _('General Ledger') }}</h1>
           </tr>
           <tr>
               <td>
-                <p>{{ _("When the Move number is between '()' means it hasn't Post Number and the shown number is the provisional one.") }}</p>
+                <p>{{ _("When the Move number is between '()' means it hasn't Post Number and the shown number is the provisional one.") }}<br/>
+                {{ _("When the Move Lines header is in read means it's not correct the Move Lines.") }}</p>
```

Comment:
I guess you meant "red" not "read"?

I think a better wording could be:

```
When move lines header are in red it means that the move lines are not correct.
```

Reply:
But as stated before, we should not really change the color nor add this comment.